changelog.md

```diff
@@ -14,10 +14,12 @@
   `ClearDirection` and reapply directions where appropriate using `SetDirection`.
 * `FindWithUsagePoints` was deemed too use-case specific for the SDK and has been removed.
 * Removal of deprecated `Terminal.tracedPhases` property. Use `Terminal.normalPhases` and `Terminal.currentPhases` instead.
+* Renamed `BaseDatabaseWriter::saveSchema` to `BaseDatabaseWriter::populateTables`.
 
 ### New Features
 * Added `ClearDirection` that clears feeder directions.
 * Added new `FeederDirection.CONNECTOR` value for `Connector` equipment that are modelled only with a single terminal.
+* Added `MetricsDataSourceWriter`, which can be used to write metrics to a Postgres database.
```

Comment:
todo: Can be used to save to more than just a Postgres database. Update this appropriately based on changes as per @gcarwardine suggestions